package.json

```diff
@@ -2,7 +2,7 @@
   "name": "nordcraft",
   "type": "module",
   "license": "Apache-2.0",
-  "version": "1.0.10",
+  "version": "1.0.11",
```

Comment:
Is it important enough to bump the version for this change?

Reply:
This is a fix I need for my current task in our internal project. So bumping the version will allow me to continue and merge that work without waiting for enough changes to be accumulated for us to bump the version.